database/schema.sql

```diff
@@ -5,3 +5,24 @@ set client_min_messages to warning;
 drop schema "public" cascade;
 
 create schema "public";
+
+CREATE TABLE "public"."restaurants" (
+	"id"          serial   NOT NULL,
+	"name"        text     NOT NULL,
+	"location"    text     NOT NULL,
+	"priceRange"  int      NOT NULL check("priceRange" >= 1 and "priceRange" <= 5),
+  PRIMARY KEY ("id")
+);
+
+
+
+CREATE TABLE "public"."reviews" (
+	"id"            serial   NOT NULL,
+	"restaurant.id" int   NOT NULL,
```

Comment:
This name should match the name in the restaurants table: "restaurantId".
I thought I had suggested this change, also. Though maybe I'm remembering someone else's schema.